std/traits.d

```diff
@@ -2718,7 +2714,7 @@ alias FieldTypeTuple = Fields;
     static assert(is(FieldTypeTuple!NestedClass == AliasSeq!int));
 
     static interface I {}
-    static assert(is(Fields!I == AliasSeq!()));
+    static assert(is(Fields!I == AliasSeq!(I)));
```

Comment:
> Fix unittest

This is a regression... The Druntime implementation was not updated alongside phobos

Reply:
Fixes: https://github.com/dlang/druntime/pull/3569 ; https://github.com/dlang/phobos/pull/8241 .

Will re-add the test to phobos once druntime is fixed.

Reply:
This is yet another piece of evidence that code duplication is bad.